oaknational/Oak-Web-Application
# feat: brush borders

## Description

- Adds BrushBorder component. 
Builds on Matt's work on the button borders, BrushBorders takes color prop to match parent component and optional 'hideOnMobile' prop to hide horizontal brush borders when a component is full width on mobile. This is to stop unwanted margins
- Adds storybook for BrushBorders
- Adds component to pages from designs


## Issue(s)

Fixes #

## How to test

1. Go to all pages with brush borders
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
